tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "checkJs": false,
-    "removeComments": false
```

Comment:
Why are we removing this?

Reply:
It's accidental, but it already defaults to false